src/fsharp/fsi/fsi.fs

```diff
@@ -2789,9 +2821,9 @@ module Settings =
                      setSignal exitSignal
         interface System.IDisposable with 
              member x.Dispose() =
-                         runSignal.Close();
-                         exitSignal.Close();
-                         doneSignal.Close();
+                 runSignal.Dispose();
+                 exitSignal.Dispose();
+                 doneSignal.Dispose();
```

Comment:
Not sure what these indentation changes are for - is it aligning code with Microsoft\visualfsharp? If so that's good.